### PR TITLE
[API-672] Edit arrays as newline-separated textarea

### PR DESCRIPTION
**Pull request by billkauf**
https://pagerduty.atlassian.net/browse/API-672

Edit simple (non-list, non-password) array parameters as `textarea`s, where each line is a separate value.

image

### CODE REVIEW

File: src/main/template/param.handlebars [hunk 1]
```diff
@@ -29,15 +29,15 @@
             </label>
             <div class="parameter-content-type"/>
         {{else}}
-            {{#if default}}
-                <input class='parameter form-control' minlength='0' name='{{name}}' placeholder='' type='{{inputType}}' value='{{default}}'  data-toggle="tooltip" data-placement="right" title="" data-original-title='{{name}}'/>
+            {{#if isArray}}
+                <textarea class='parameter form-control array-parameter' minlength='0' name='{{name}}' placeholder='' type='{{inputType}}' data-toggle="tooltip" data-placement="right" title="" data-original-title='{{name}}'>{{default}}</textarea>
```
Comment (brhou) [inline src/main/template/param.handlebars]: Can we change `placeholder` to:

screen shot 2016-02-23 at 5 21 14 pm

File: src/main/template/param_required.handlebars [hunk 2]
```diff
@@ -16,16 +16,16 @@
         {{#if isFile}}
             <input class='parameter form-control required' type='file' name='{{name}}'/>
         {{else}}
-            {{#if default}}
-                <input class='parameter form-control required' minlength='1' name='{{name}}' placeholder='(required)' type='{{inputType}}' value='{{default}}' data-toggle="tooltip" data-placement="right" title="" data-original-title='{{name}}'/>
+            {{#if isArray}}
+                <textarea class='parameter form-control required array-parameter' minlength='1' name='{{name}}' placeholder='(required)' type='{{inputType}}' data-toggle="tooltip" data-placement="right" title="" data-original-title='{{name}}'>{{default}}</textarea>
```
Comment (brhou) [inline src/main/template/param_required.handlebars]: https://github.com/PagerDuty/swagger-ui/pull/8/files#r53880927

same here

File: src/main/template/param.handlebars (hunk 1 above)
Comment (brhou) [inline src/main/template/param.handlebars]: sorry, didn't see this https://github.com/PagerDuty/swagger-ui/pull/8/files#diff-5e8d21826da50b308155018758e5f116R40. Either way works.

**Comment by billkauf** [comment]
OK, but it doesn't look too hot:

image

**Comment by brhou** [comment]
Actually, I think it looks fine, and we probably want to remove the `one per line` text, since that area is meant to describe the type of `user_ids[]`, not the text field usage.

**Comment by brhou** [comment]
:ship:

**Pull request merged**